### PR TITLE
Added additional tests using enzyme

### PR DESCRIPTION
**Pull request by grezzzy**

### CODE REVIEW

**Comment by a-nouraldean** [comment]
I would not worry about fixing this conflict. Nice job reverting to react16 
https://github.com/enzymejs/enzyme/pull/2430

File: client/components/Accordion.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React, { useState, useRef, useEffect } from 'react';
 import ReactDOM from 'react-dom';
 import Chevron from './chevron.jsx';
 
-const Accordian = (props) => {
+const Accordion = (props) => {
```
Comment (a-nouraldean) [inline client/components/Accordion.jsx:5]: Nice catch!

File: public/styles.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
-.accordion-section {
-  width: 30%;
-}
+/* .accordion-section {
+  width: 25%;
```
Comment (a-nouraldean) [inline public/styles.css:2]: Does it make sense to use the CSS resize property?

https://developer.mozilla.org/en-US/docs/Web/CSS/resize

**Review by a-nouraldean — APPROVED** [review]
Nice work! Really cool seeing you project comment. Just had two small comments.

**Review by a-nouraldean — APPROVED** [review]

File: public/styles.css (hunk 2 above)
Comment (grezzzy) [reply public/styles.css:2]: I will check into this. Wanted to check in with the team about styling before building proxy servers

**Pull request merged**